content/it/community/_index.html

```diff
@@ -1,69 +1,237 @@
-            </div>
-            <div>
-                <a href="http://slack.k8s.io/">#kubernetes-users</a>
-                <p>Il nostro canale Slack è il modo migliore per contattare i nostri ingegneri e condividere le tue idee con loro.</p>
-            </div>
-            <div>
-                <a href="http://stackoverflow.com/questions/tagged/kubernetes">Stack Overflow</a>
-                <p>Il nostro forum di utenti è un ottimo posto dove andare per il supporto della comunità.</p>
-            </div>
-        </div>
-    </main>
-</section>
+<div class="newcommunitywrapper">
+<div class="banner1">
+  <img src="/images/community/kubernetes-community-final-02.jpg" alt="Kubernetes Conference Gallery" style="width:100%" class="desktop">
+  <img src="/images/community/kubernetes-community-02-mobile.jpg" alt="Kubernetes Conference Gallery" style="width:100%" class="mobile">
+</div>
+
+<div class="intro">
+<br class="mobile">
+<p>La comunità di Kubernetes - cioè gli utenti, i contributori e la cultura che abbiamo costruito tutti assieme - è una dei maggiori motivi della crescita esponenziale di questo progetto open-source. La nostra cultura e i nostri valori continuano a crescere e modificarsi con il progetto stesso. Lavoriamo tutti assieme per il miglioramento costante del progetto e nei modi di interazione.
+<br><br>Noi siamo le persone che aprono gli issue, le pull request, che partecipano alle riunioni delle SIG, i meetup di Kubernetes, e le KubeCon, che spingiamo l'innovazione e l'adozione di Kubernetes, che eseguiamo <code>kubectl get pods</code>, e che contribuiscono in moltissimi altri modi egualmente vitali al progetto. Vai avanti a leggere per scoprire come essere coinvolto in queste attività e diventare parte di questa meravigliosa comunità.</p>
+<br class="mobile">
+</div>
+
+<div class="navbar">
+
+<a href="#conduct">Codice di Condotta</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+<a href="#videos">Video</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+<a href="#discuss">Discussioni</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+<a href="#events">Eventi e meetups</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+<a href="#news">Notizie</a>
+
+</div>
+<br class="mobile"><br class="mobile">
+<div class="imagecols">
+<br class="mobile">
+  <div class="imagecol">
+    <img src="/images/community/kubernetes-community-final-03.jpg" alt="Kubernetes Conference Gallery" style="width:100%" class="desktop">
+  </div>
+
+  <div class="imagecol">
+    <img src="/images/community/kubernetes-community-final-04.jpg" alt="Kubernetes Conference Gallery" style="width:100%" class="desktop">
+  </div>
+
+  <div class="imagecol" style="margin-right:0% important">
+    <img src="/images/community/kubernetes-community-final-05.jpg" alt="Kubernetes Conference Gallery" style="width:100%;margin-right:0% important" class="desktop">
+  </div>
+  <img src="/images/community/kubernetes-community-04-mobile.jpg" alt="Kubernetes Conference Gallery" style="width:100%;margin-bottom:3%" class="mobile">
```

Comment:
Optional fix: localize the `alt` attributes.

Reply:
thanks